src/Microsoft.Data.Analysis/IntDataFrameColumn.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Data.Analysis
+{
+    public partial class IntDataFrameColumn : PrimitiveDataFrameColumn<int>
```

Comment:
Int32. We should be using the `System.XXX` type names in APIs. Using the C# names (int, long, short) is confusing to other languages - i.e. F#.

Reply:
Done.